src/react/components/styleform/LineTab.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { SvgRadioButton } from 'oskari-ui';
+import { Form, Row } from 'antd';
+
+export const LineTab = (props) => {
+    return (
+        <>
+            <Row>
+                <Form.Item name='lineStyle' label='Line dash' { ...props.formLayout }>
+                    <SvgRadioButton
+                        options={ props.lineIcons.lineDash }
+                        onChange={ (event) => props.onChangeCallback('stroke.lineDash', event.target.value) }
+                        defaultValue={ props.styleSettings.stroke.lineDash }
```

Comment:
These need a bunch of checks to make sure we have the whole structure available.

Reply:
Also you might be able to just pass the line-settings from the whole style JSON to this component to make it easier for everyone to see what's happening.